apps/cookbook/src/app/extensions/extensions-landing-page.component.html

```diff
@@ -0,0 +1,37 @@
+<article>
+  <h1>Kirby Extensions for Angular</h1>
+  <p>
+    The Kirby Extensions library is a collection of design guidelines and components that build on
```

Comment:
Does it also have 'guidelines' or is it actually "just" a collection of extra components?

Reply:
It is going to contain guidelines as well 👍

Reply:
Very nice !